src/pool/srv_pool.c

```diff
@@ -1112,6 +1322,19 @@ pool_svc_step_up_cb(struct ds_rsvc *rsvc)
 		goto out;
 	event_cb_registered = true;
 
+	/* Update pool map by IV */
+	rc = pool_map_update(svc, map_version, map_buf);
```

Comment:
Line 1315 already requests an async pool map update. Could you elaborate why we need another sync pool map update here, please?

Reply:
ah, indeed, this can be removed. Thanks.